rmf_task_ros2/src/rmf_task_ros2/bidding/Auctioneer.cpp

```diff
@@ -15,17 +15,21 @@
  *
 */
 
+#include <rclcpp/create_publisher.hpp>
+#include <rclcpp/create_subscription.hpp>
+#include <rclcpp/create_timer.hpp>
```

Comment:
Can probably replace these and the `#include <rclcpp/node.hpp>` in `Auctioneer.hpp` with a single `#include <rclcpp/rclcpp.hpp>` in `Auctioneer.hpp` but keeping a minimal set of imports is fine too 👍🏼